integration_tests/test_basic.py

```diff
@@ -427,3 +428,28 @@ def test_message_call(cronos):
     assert 23828976 == receipt.cumulativeGasUsed
     assert receipt.status == 1, "shouldn't fail"
     assert len(receipt.logs) == iterations
+
+
+def test_suicide(cluster):
+    """
+    test compliance of contract suicide
+    - within the tx, after contract suicide, the code is still available.
+    - after the tx, the code is not available.
+    """
+    w3 = cluster.w3
+    inner = deploy_contract(
```

Comment:
what inner and outer means?

Reply:
they are just the contract names: https://github.com/crypto-org-chain/cronos/pull/286/files#diff-691042267b4f7caaa41185a26ffad9fae48b8960f4c668bba805f4cb782f527fR4
inner contract is to be destroyed by the outer contract.

Reply:
I assumed that after reading through it, but was puzzled by the names as well 😂 perhaps a comment there or renaming it as "to_be_destroyed"/"destroyee" and "destroyer" contracts?

Reply:
sounds good, renamed.